internal/integration/bounce_rule_test.go

```diff
@@ -0,0 +1,211 @@
+package integration
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/go-sql-driver/mysql"
+
+	"github.com/jimmyjames85/bouncecm/internal/models"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type BounceRuleSuite struct {
+	suite.Suite
+
+	handler http.HandlerFunc
+	client  *http.Client
+}
+
+func (suite *BounceRuleSuite) SetupSuite() {
+	suite.client = &http.Client{}
+}
+
+func (suite *BounceRuleSuite) SetupTest() {
+	_, err := Database.Exec(`
+		CREATE TABLE IF NOT EXISTS bounce_rule (
+			id smallint(5) unsigned NOT NULL AUTO_INCREMENT,
+			response_code smallint(5) unsigned NOT NULL DEFAULT '0',
+			enhanced_code varchar(16) NOT NULL DEFAULT '',
+			regex varchar(255) NOT NULL DEFAULT '',
+			priority tinyint(3) unsigned NOT NULL DEFAULT '0',
+			description varchar(255) DEFAULT NULL,
+			bounce_action varchar(255) NOT NULL,
+			PRIMARY KEY (id),
+			UNIQUE KEY bounce_rule_components (response_code,enhanced_code,regex)
+		) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+	`)
+
+	if err != nil {
+		suite.T().Fatalf("Failed to setup for test\nError: %v", err)
+	}
+
+	mysql.RegisterLocalFile("testdata/bounce_rules.csv")
+
+	res, err := Database.Exec("LOAD DATA LOCAL INFILE '" + "testdata/bounce_rules.csv" + "' INTO TABLE bounce_rule FIELDS TERMINATED BY ',' LINES TERMINATED BY '\n'")
+	if err != nil {
+		suite.T().Fatalf("Failed to load data from file\nError: %v", err)
+	}
+	inserted, err := res.RowsAffected()
+	if err != nil {
+		suite.T().Fatalf("Failed to get inserted rows\nError: %v", err)
+	}
+	if inserted != 295 {
```

Comment:
I wouldn't do this check, otherwise you're going to have to update it every time if you add more things to the bounce rules csv. I recommend just checking to make sure `inserted > 0` so you know that it worked